packages/phishing-controller/src/PhishingController.ts

```diff
@@ -819,6 +819,22 @@ export class PhishingController extends BaseController<
     return combinedResponse;
   };
 
+  /**
+   * Get a cached result if it exists and is not expired
+   *
+   * @param url - The URL to get the cached result for. You can pass in a full URL or just the hostname.
+   * @returns The cached scan result or undefined if not found or expired
+   */
+  getScanResultFromCache = (
```

Comment:
We should not need this method at all... any caller to the phishing controller API can just rely on the scanUrl method that is already checking the internal cache before doing any network request to the real time dapp scanning API?